src/model/layer/pooling.cc

```diff
@@ -85,15 +85,15 @@ const Tensor Pooling::Forward(int flag, const Tensor& input) {
     Tensor mask;
     mask.ResetLike(output);
     float* maskptr = new float[mask.Size()];
-    ForwardMaxPooling(inptr, batchsize, channels_, height_, width_, kernel_h_,
-                      kernel_w_, pad_h_, pad_w_, stride_h_, stride_w_, outptr,
+    ForwardMaxPooling(inptr, (int)batchsize, (int)channels_, (int)height_, (int)width_, (int)kernel_h_,
```

Comment:
it is not a good solution to do type cast in this way.
maybe we can add this in singa_config.h
```
#if wind32
typedef size_t unsigned int
#endif
```